dev-docs/bidder-adaptor.md

```diff
@@ -495,9 +495,102 @@ if (bid.mediaType === 'video' || (videoMediaType && context !== 'outstream')) {
 
 #### Long-Form Video Content
 
-To support long-form videos it is the responsibility of the adapter to convert their categories into [IAB accepted subcategories]( http://iabtechlab.com/wp-content/uploads/2017/11/IAB_Tech_Lab_Content_Taxonomy_V2_Final_2017-11.xlsx) (links to MS Excel file). Each bid request must return one IAB subcategory.
+{: .alert.alert-info :}
+Following is Prebid's way to setup bid request for long-form. Apadters are free to choose their own approach.
+
+Prebid now accepts multiple bid responses for a single `bidRequest.bids` object. For each Ad pod Prebid expects you to send back n bid responses. It is up to you how you send back bid responses. Prebid's recommendation is that you expand an adpod placement into a set of request objects according to total adpod duration and the range of duration seconds. It also depends on your endpoint as well how you may want to create your request for long-form. Appnexus adapter follows below algorithm to expand its placement. 
+
+Use case 1
+```
+AdUnit config
+{
+  ...
+  adPodDuration: 300,
+  durationRangeSec: [15, 30]
+  ...
+}
+
+Algorithm
+# of placements = adPodDuration / MIN_VALUE(durationRangeSec)
+
+Each placement set max duration:
+placement.video.maxduration = MAX_VALUE(durationRangeSec)
+
+Example:
+# of placements : 300 / 15 = 20.
+placement.video.maxduration = 30 (all placements the same)
+
+Your endpoint responds with:
+10 bids with 30 seconds duration
+10 bids with 15 seconds duration
+```
+
+In Use case 1, you are asking endpoint to respond with 20 bids between min duration 0 and max duration 30 seconds. If you get bids with duration which does not match duration in `durationRangeSec` array, Prebid will modify the duration and use new value to send bids to Ad server.
```

Comment:
To clarify - we aren't modifying the actual duration in the actual bid.  We're evaluating the bid's duration and will match into the appropriate duration bucket by using a rounding-type logic.